test/docs/OpenJ9TestUserGuide.md

```diff
@@ -0,0 +1,302 @@
+<!--
+Copyright (c) 2016, 2018 IBM Corp. and others
+
+This program and the accompanying materials are made available under
+the terms of the Eclipse Public License 2.0 which accompanies this
+distribution and is available at https://www.eclipse.org/legal/epl-2.0/
+or the Apache License, Version 2.0 which accompanies this distribution and
+is available at https://www.apache.org/licenses/LICENSE-2.0.
+
+This Source Code may also be made available under the following
+Secondary Licenses when the conditions for such availability set
+forth in the Eclipse Public License, v. 2.0 are satisfied: GNU
+General Public License, version 2 with the GNU Classpath
+Exception [1] and GNU General Public License, version 2 with the
+OpenJDK Assembly Exception [2].
+
+[1] https://www.gnu.org/software/classpath/license.html
+[2] http://openjdk.java.net/legal/assembly-exception.html
+
+SPDX-License-Identifier: EPL-2.0 OR Apache-2.0 OR GPL-2.0 WITH Classpath-exception-2.0 OR LicenseRef-GPL-2.0 WITH Assembly-exception
+-->
+
+# How-to Run Tests
+
+Let's create the following scenario:
+For the purpose of this example, we assume you have an OpenJ9 SDK ready
+for testing. Below are the specific commands you'd run to compile and
+run tests. Details are explained in *Tasks in OpenJ9 Test* section below.
+
+```
+cd openj9/test/TestConfig
+export JAVA_BIN=<path to JDK bin directory that you wish to test>
+export SPEC=linux_x86-64_cmprssptrs
+make -f run_configure.mk
+make test
+```
+
```

Comment:
Probably should change this example to be the more common, run sanity tests...  (adding make compile and make _sanity, as I think less and less folks would run "make test"